specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2019-08-01-preview/OperationalInsights.json

```diff
@@ -637,22 +637,22 @@
       "in": "path",
       "required": true,
       "type": "string",
-      "description": "The workspace's resource subscription ID."
+      "description": "Gets subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
     },
     "ApiVersionParameter": {
       "name": "api-version",
       "in": "query",
       "required": true,
       "type": "string",
-      "description": "The client API version."
+      "description": "Client Api Version."
```

Comment:
`The client API version` is better than `Client Api Version` to me

Reply:
Make sense. Let me change this